erpnext/templates/pages/order.html

```diff
@@ -75,7 +79,7 @@ <h1>{{ doc.name }}</h1>
 		<p class="small" style="float: right;">
 			<b>{{_("Delivery Date")}}</b>
 			<br>
-			{{ doc.get_formatted("delivery_date") }}
+			{{ frappe.utils.global_date_format(doc.get_formatted("delivery_date")) }}
```

Comment:
What is the point of this change?

Reply:
You told me to show delivery date like March 20,2020 so I change it

Reply:
Use `{{ frappe.utils.global_date_format(doc.delivery_date) }}` instead... `global_date_format` does not expect formatted date as an input

Reply:
Alright